tests/complete_run_v2_3_0.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import os
+from PIL import Image, ImageChops
+from system.test_diags import compare_images
+import sys
+sys.path.insert(0, '../examples')
+from tutorials.tutorial_v2_3_0_all_sets import run_anvil
+
+# Due to the large amount of data required to run, this test will be run manually on Anvil
```

Comment:
Since this script will be run manually, it will be handy to add instruction on running on compute nodes and activating e3sm-unified env.